_layouts/default.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   	<head>
-  		<title>Strata by HTML5 UP</title>
+  		<title>{{ site.title }}</title>
```

Comment:
This should be page.title by default and fall back to site.title so the page titles aren't the same on all pages